docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -42,6 +42,9 @@ For these platforms, the platform alone determines the browser that is used.
 |iOS|any|Safari with WKWebView|
 |Android|any|Chrome|
 
+> [!IMPORTANT]
+> [Conditional Access](/azure/active-directory/conditional-access/overview) is not supported for Office Add-ins on iOS. Those add-ins use embedded Safari, not Edge.
```

Comment:
When I click the link in the preview, I get this URL: https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ConditionalAccess/azure/active-directory/conditional-access/overview
I assume this works once the change is published?

Reply:
Yeah, that's a publishing thing. It'll go here: https://learn.microsoft.com/en-us/azure/active-directory/conditional-access/overview